pkg/eks/api.go

```diff
@@ -248,7 +248,6 @@ func newSession(clusterConfig *api.ClusterConfig, endpoint string, credentials *
 		}
 	}
 
-	logger.Info("using region %s", clusterConfig.Region)
```

Comment:
👍 - I noticed this as part of my previous change but decided to leave it out as I wasn't really sure if it was relevant for other calls. Seems like it is when we are actually creating the cluster.